### PR TITLE
flag hidden files

### PR DESCRIPTION
**Pull request by Skill-Zozo**
hidden files are normally being used by other programs, probably best to not mess with them unless the user gives the go ahead

### CODE REVIEW

File: pysorter/core/sorter.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def __init__(self,
                  sort_rule,
                  no_process=None,
                  dest_dir=None,
-
+                 do_move_hiddens=False,
```
Comment (chriscz) [inline pysorter/core/sorter.py]: do_move_hidden please ;)

File: pysorter/core/pysorter.py [hunk 2]
```diff
@@ -68,6 +68,11 @@ def parse_args(args=None):
                         action='store_true',
                         dest='do_recurse')
 
+    parser.add_argument('U', '--move_hidden_files',
```
Comment (chriscz) [inline pysorter/core/pysorter.py]: replace `_`'s with `-`'s

**Comment by Skill-Zozo** [comment]
Okay, this maybe me not being a pysorter power user but I haven't figured out a way to skip hidden files using the rules in filetypes. But I think pysorter is normally for visual effect you wanna see your directory all sorted and what not, one does not exactly care about files you can't see. Also I have an pysorter on other machines and people complain that it moved hidden files and screwed around with their configs for other programs

**Comment by chriscz** [comment]
The community should eventually decide whether or not they want the feature, since your argument is valid, I'll merge it after you make the suggested changes.

**Comment by chriscz** [comment]
Closing due to non-response

**Pull request closed without merging**